src/os/mainctrl.js

```diff
@@ -273,6 +274,8 @@ os.MainCtrl = function($scope, $element, $compile, $timeout, $injector) {
       'Clear all spatial query areas'));
   os.ui.clearManager.addEntry(new os.ui.clear.ClearEntry('states', 'States', os.ui.state.cmd.StateClear,
       'Deactivate all states'));
+  os.ui.clearManager.addEntry(new os.ui.clear.ClearEntry('places', 'Places/Annotations', plugin.places.PlacesHide,
```

Comment:
The Places plugin should be adding this in its `init` method, so it doesn't show up in the Clear menu when the plugin is disabled.